testHelpers/utilsBuilder.ts

```diff
@@ -45,7 +46,9 @@ class UtilsBuilder {
       VoucherKernel: await ethers.getContractFactory('VoucherKernel'),
       Cashier: await ethers.getContractFactory('Cashier'),
       BosonRouter: await ethers.getContractFactory('BosonRouter'),
-      ERC1155ERC721: await ethers.getContractFactory('ERC1155ERC721'),
+      //ERC1155ERC721: await ethers.getContractFactory('ERC1155ERC721'),
```

Comment:
Please remove this line.

Reply:
Removed. Will be included in next commit.